packages/react-performance/src/performance-report.ts

```diff
@@ -11,7 +11,10 @@ export interface ErrorHandler {
 
 export function usePerformanceReport(
   url: string,
-  {onError = noop}: {onError?: ErrorHandler} = {},
+  {
+    locale = undefined,
```

Comment:
I'm on the fence about whether this should be a more free-form `metadata`-like property that accepts/transmits app-specific custom tags.

If more values get crammed in here, I'd be happy to rework to make this more generic in a later release 😅